packages/drivers/odsp-driver/src/odspDocumentDeltaConnection.ts

```diff
@@ -633,12 +634,20 @@ export class OdspDocumentDeltaConnection extends DocumentDeltaConnection {
 			}
 		});
 
-		await super.initialize(connectMessage, timeout).finally(() => {
+		try {
```

Comment:
We should remove this entirely as supportedFeatures are features that ordering service supports and those should just be readonly and should not be modified.